src/predict/gcp_helpers/cloud_function.py

```diff
@@ -16,12 +16,17 @@
 from pymongo import MongoClient
```

Comment:
For easy maintainability, we could move the code to its own microservice. Details in this PR: https://github.com/airqo-platform/AirQo-api/pull/431.

How are we handling `tenancy` and `envs`?